builder/test/tests.sh

```diff
@@ -37,7 +37,6 @@ roscore -h
 rosversion clever
 rosversion aruco_pose
 rosversion vl53l1x
-rosversion opencv3
```

Comment:
Why there is no test for opencv presents in the image?

Reply:
`opencv3` is no longer a ROS package in Melodic, so this will always fail (as it should). The proper test for opencv version is performed later, in `tests.py`, where OpenCV build information (along with its version) is printed.